src/libbiosiglite/biosig4c++/biosig.c

```diff
@@ -82,7 +82,9 @@ int VERBOSE_LEVEL __attribute__ ((visibility ("default") )) = 0;	// this variabl
 #define max(a,b)        (((a) > (b)) ? (a) : (b))
 
 char* getlogin (void);
-char* xgethostname (void);
+char* xgethostname (void){
```

Comment:
Link error since this function was not defined previously. Not sure what should be default return value from this function. Please have a closer look at this change.